src/Game/Managers/PartyManager.cs

```diff
@@ -53,68 +53,139 @@ public void ParsePacket(Packet p)
                     add = true;
                     goto case 2;
                 case 2:
-                    byte count = p.ReadByte();
 
-                    if (count <= 1)
+                    if (Engine.Profile.Current.CustomBarsToggled == true)
```

Comment:
Is there any specific reason to have a code duplicate?
I mean you can write the party stuff 1 time and inside check for CustomHB or StandardHB